src/java/org/apache/cassandra/locator/IEndpointSnitch.java

```diff
@@ -30,12 +30,12 @@
 public interface IEndpointSnitch
 {
     /**
-     * returns a String repesenting the rack this endpoint belongs to
+     * returns a String representing the rack this endpoint belongs to
      */
     public String getRack(InetAddress endpoint);
 
     /**
-     * returns a String representing the datacenter this endpoint belongs to
+     * returns a String representing the data center this endpoint belongs to
```

Comment:
I don't think this is warranted.
 `datacenter` is the more popular term within the repository than `data center`

```
cassandra-git (trunk) $ find . -type f -exec grep -l "datacenter" {} \; | wc -l
98
cassandra-git (trunk) $ find . -type f -exec grep -l "data center" {} \; | wc -l
56
```